x-pack/plugins/index_management/public/application/components/component_templates/__jest__/client_integration/component_template_list.test.ts

```diff
@@ -166,7 +166,7 @@ describe('<ComponentTemplateList />', () => {
 
       expect(exists('componentTemplatesLoadError')).toBe(true);
       expect(find('componentTemplatesLoadError').text()).toContain(
-        'Unable to load component templates. Try again.'
+        'Error loading component templatesInternal server error'
```

Comment:
It seems that a space and a dot is missing after "templates".

Reply:
Nice spot! This is not a bug though, just an oddity due to the way the DOM API provides `text` values of child elements by concatenating each child element's text value without any delimiter. So it's smashing together the title element "Error loading component templates" with the description element "Internal server error".